models/classes/runner/QtiRunnerService.php

```diff
@@ -1073,7 +1074,10 @@ public function getItemSession(RunnerServiceContext $context)
         // The current item answered state
         $route    = $session->getRoute();
         $position = $route->getPosition();
-        $output['itemAnswered'] = TestRunnerUtils::isItemCompleted($route->getRouteItemAt($position), $itemSession);
+        $config = $this->getTestConfig();
+        $reviewConfig = $config->getConfigValue('review');
+        $partiallyAnsweredIsAnswered = isset($reviewConfig['partiallyAnsweredIsAnswered']) ? (bool) $reviewConfig['partiallyAnsweredIsAnswered'] : true;
```

Comment:
Is it possible for `$reviewConfig` to be undefined/null? If yes, should check for it.

Reply:
I see it's not checked in [other places](https://github.com/oat-sa/extension-tao-testqti/blob/develop/models/classes/runner/map/QtiRunnerMap.php#L159),  so ignore it